### PR TITLE
Recipe_app: Release v1.1

### PR DESCRIPTION
**Pull request by fherrerao**
# Hi There 💯 

Here is the link to [video demonstration](https://www.loom.com/share/113e5029c7d2458aa0b0069277ae79a2).

Features implemented in this project:

### Setup and tools used

- Set up model associations, following this [ER Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/images/erd_diagram.png)
- Add validations for all developed models.
- Add devise gem to allow users to sign up and sign in and handle sessions in the app.
- Add cancancan gem to manage authorization for all corresponding actions.

### Views, templates and layouts

- Add splash screen with the name of the app and links to the sign up and log in pages.
- When the user logs in, they are presented with the categories page.
- For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
- There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
- A new category form to add a new one.
- A new transaction form to add a new one.
- At the top of the Transaction page the user could see the total amount for the category

### Quality Assurance
- Add unit tests in the spec/models for all models.
- Add integration tests in the spec/integration for all developed views.

Please let me know if I should make changes.

### CODE REVIEW

File: spec/models/group_spec.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  before :all do
+    User.destroy_all
+    @user = User.new(name: 'Kasuki', email: 'first@email.com', password: '123456',
+                     password_confirmation: '123456')
+    @user.skip_confirmation!
```
Comment (TedLivist) [inline spec/models/group_spec.rb]: - [x] All your tests seem to be failing due to this method (for example I could not generate the seed data until I removed this line as well). However, even when I remove this line the tests still fail anyway. I would recommend you fix this

  herrera

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @fherrerao,

Good job so far!
✔️ Nice UI design
✔️ Implementation of validations
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/group_spec.rb (hunk 1 above)
Comment (fherrerao) [reply spec/models/group_spec.rb]: Oh sorry, I made some changes, and I forgot to fix the tests. 😵‍💫, I just corrected those issues, and now they are working.

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Linters were passing on my local machine as well**

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**